packages/nimble-components/src/wafer-map/tests/wafer-map.stories.ts

```diff
@@ -135,7 +135,7 @@ const metadata: Meta<WaferMapArgs> = {
         highlightedValues: 'set1',
         maxCharacters: 4,
         orientation: WaferMapOrientation.left,
-        quadrant: WaferMapQuadrant.bottomLeft
+        originLocation: WaferMapOriginLocation.bottomLeft
```

Comment:
Okay, so despite the behavior change in the attribute, it makes sense to keep this the same (as the default), since it matches the true default of the attribute (which it didn't before)?

Reply:
yes, I think before we went with a top left origin similar to graphics rendering origin. I do not remember why the storybook was using this default though